lib/barbeque/worker.rb

```diff
@@ -33,34 +31,35 @@ def self.run(
     end
 
     def initialize
-      @queue_name = ENV['BARBEQUE_QUEUE'] || DEFAULT_QUEUE
+      @command =
+        case worker_id
```

Comment:
I'm trying this runner + poller architecture in Kuroko2 on ahead in company (will be published after testing with our many jobs), and noticed that the runner might have trouble (e.g., being stuck, taking much time) when Hako has some bugs or faces some unexpected situation while the poller has no trouble for now.
From this experience, I think barbeque-worker should have multiple runners with single poller.

The metrics for poller can be approximated by `job_executions.created_at - job_executions.finished_at`. Smaller is better.
The metrics for runner looks a bit harder. Adding `enqueued_at` field to SQS message body?